### ISSUE TITLE
False positive when local module has same name of a workspace.

### ISSUE THREAD

**Issue by FezVrasta** [joshuajaco/eslint-plugin-workspaces#32]
Hi, I have this structure:

```
- workspaces
  - foo
  - bar
    - foo
    index.ts
```

When I import `./foo` from `index.ts`, it complains with `eslint[workspaces/require-dependency]` thinking I'm trying to import the workspace foo, while I'm just trying to import the local `foo` module.

**Comment by joshuajaco** [comment]
Thanks for logging this. Fix is released in `0.10.1`

**Comment by FezVrasta** [comment]
Thanks!
